javascript/javascript2/week1/shortestWordredo.js

```diff
@@ -0,0 +1,7 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function getShortestWord(danishWords) {
+    return danishWords.find(word => word.length === Math.min(...(danishWords.map(word => word.length))));
+}
```

Comment:
Cool solution! Though I would rewrite it as this:
```
function getShortestWord(danishWords) {
  const wordLengths = danishWords.map(word => word.length);
  const minWordLength =  Math.min(...wordLengths);
  return danishWords.find(word => word.length === minWordLength);
}
```

Both to make it easier to read but also to not re do the min and mapping operations in each filter step.